README.md

```diff
@@ -12,6 +12,10 @@ earlier Windows versions. Growl is used if none of these requirements are met.
 
 ![Input Example](https://raw.githubusercontent.com/mikaelbr/node-notifier/master/example/input-example.gif)
 
+## Actions Example Windows SnoreToast
+
+![Actions Example](https://raw.githubusercontent.com/yoavain/node-notifier/master/example/windows-actions-example.gif)
```

Comment:
Thanks for adding a link to the example! We'll apply this change before merging to have it point to the main repo, since the link won't work until this PR gets merged but we don't want to link to your fork :)
```suggestion
![Actions Example](https://raw.githubusercontent.com/mikaelbr/node-notifier/master/example/windows-actions-example.gif)
```